filebeat/inputsource/unix/server_test.go

```diff
@@ -60,6 +60,10 @@ func TestErrorOnEmptyLineDelimiter(t *testing.T) {
 }
 
 func TestReceiveEventsAndMetadata(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("test is only supported on non-windows")
```

Comment:
Do we need the same comment as in https://github.com/elastic/beats/pull/21750/files#diff-f78953ffe73980d156eb9bc06aa6cbaa4e32d81dd629d2d37af441c585934ef0R263?

Reply:
```suggestion
		t.Skip("test is only supported on non-windows. See https://github.com/elastic/beats/issues/19641")
```